### PR TITLE
TECH-976 Test and deploy events map changes

### PR DESCRIPTION
**Pull request by cjmabry**

### CODE REVIEW

File: scss/events-map.scss [hunk 1]
```diff
@@ -581,7 +585,7 @@ article#events {
         ul {
           li {
             display: inline-block;
-            width: calc(50% - 20px);
+            // width: calc(50% - 20px);
```
Comment (ericbroder) [inline scss/events-map.scss:588]: is this intentional?

File: scss/events-map.scss [hunk 2]
```diff
@@ -512,7 +512,7 @@ article#events {
                 margin-right: 4px;
               }
 
-              .map-red-circle, .circle-button {
+              .map-red-circle, .map-purple-circle, .circle-button {
```
Comment (ericbroder) [inline scss/events-map.scss:515]: seems like the class here should just be `map-circle` or `circle-button`? and then the colors can just be subclasses or whatever?

**Review by ericbroder — APPROVED** [review]

**Comment by ericbroder** [comment]
A couple minor comments, otherwise looks good 👍

File: scss/events-map.scss (hunk 1 above)
Comment (cjmabry) [reply scss/events-map.scss:588]: Yup.

File: scss/events-map.scss (hunk 2 above)
Comment (cjmabry) [reply scss/events-map.scss:515]: usually would make the effort to refactor that into a base class and sub-class but not worth effort here.

File: scss/events-map.scss [hunk 3]
```diff
@@ -525,6 +525,10 @@ article#events {
               .map-red-circle {
                 background-color: $red;
               }
+              
+              .map-purple-circle {
```
Comment (ericbroder) [inline scss/events-map.scss:529]: @cjmabry I think we need to clear cache for css on this?

**Pull request merged**